src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml

```diff
@@ -0,0 +1,202 @@
+      attribute: "ClusterRevision"
+      PICS: CDOCONC.S.Afffd
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "Read the global attribute: FeatureMap"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS:
+          CDOCONC.S.Afffc && !CDOCONC.S.F00 && !CDOCONC.S.F01 && !CDOCONC.S.F02
+          && !CDOCONC.S.F03 && !CDOCONC.S.F04 && !CDOCONC.S.F05
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label:
+          "Given CDOCONC.S.F00(MEA) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F00
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label:
+          "Given CDOCONC.S.F01(LEV) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F01
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
+
+    - label:
+          "Given CDOCONC.S.F02(MED) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F02
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x4]
+
+    - label:
+          "Given CDOCONC.S.F03(CRI) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F03
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x8]
+
+    - label:
+          "Given CDOCONC.S.F04(PEA) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F04
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x10]
+
+    - label:
+          "Given CDOCONC.S.F05(AVG) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F05
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x20]
+
+    - label: "Read the global attribute: AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb
+      response:
+          constraints:
+              type: list
+              contains: [9, 65528, 65529, 65530, 65531, 65532, 65533]
+
+    - label: "Read the optional attribute Uncertainty in AttributeList"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      PICS: CDOCONC.S.Afffb && CDOCONC.S.A0007 && CDOCONC.S.F00
+      response:
+          constraints:
+              type: list
+              contains: [7]
```

Comment:
We should be testing that the list does not include 7 if CDOCONC.S.A0007  is not set, no?

Reply:
Thanks, added an excludes test for this too.